test/event_handler_test.py

```diff
@@ -62,38 +62,42 @@ def tearDown(self):
 
     def test_reads_messages_from_queue_with_key_from_s3(self):
         self.__setup_s3()
+        event_id_3 = str(uuid.uuid4())
+        event_id_4 = str(uuid.uuid4())
         self.__encrypt_and_send_to_sqs(
             [
                 create_event_string('sample-id-1', 'session-id-1'),
                 create_event_string('sample-id-2', 'session-id-2'),
-                create_fraud_event_string('sample-id-3', 'session-id-3', 'fraud-event-id-1'),
-                create_fraud_event_string('sample-id-4', 'session-id-4', 'fraud-event-id-2'),
+                create_fraud_event_string(event_id_3, 'session-id-3', 'fraud-event-id-1'),
+                create_fraud_event_string(event_id_4, 'session-id-4', 'fraud-event-id-2'),
             ]
         )
 
         event_handler.store_queued_events(None, None)
 
         self.__assert_audit_events_table_has_billing_event_records([('sample-id-1', 'session-id-1'), ('sample-id-2', 'session-id-2')], MINIMUM_LEVEL_OF_ASSURANCE)
-        self.__assert_audit_events_table_has_fraud_event_records([('sample-id-3', 'session-id-3', 'fraud-event-id-1'), ('sample-id-4', 'session-id-4', 'fraud-event-id-2')])
+        self.__assert_audit_events_table_has_fraud_event_records([(event_id_3, 'session-id-3', 'fraud-event-id-1'), (event_id_4, 'session-id-4', 'fraud-event-id-2')])
         self.__assert_billing_events_table_has_billing_event_records([('session-id-1', 'sample-id-1'), ('session-id-2','sample-id-2')])
-        self.__assert_fraud_events_table_has_fraud_event_records([('session-id-3', 'fraud-event-id-1'), ('session-id-4', 'fraud-event-id-2')])
+        self.__assert_fraud_events_table_has_fraud_event_records([(event_id_3, 'session-id-3', 'fraud-event-id-1'), (event_id_4, 'session-id-4', 'fraud-event-id-2')])
         self.assertEqual(self.__number_of_visible_messages(), '0')
         self.assertEqual(self.__number_of_hidden_messages(), '0')
 
     def test_reads_fraud_events_from_queue(self):
         self.__setup_s3()
+        event_id_1 = str(uuid.uuid4())
+        event_id_2 = str(uuid.uuid4())
         self.__encrypt_and_send_to_sqs(
             [
-                create_fraud_event_string('sample-id-1', 'session-id-1', 'fraud-event-id-1'),
-                create_fraud_event_string('sample-id-2', 'session-id-2', 'fraud-event-id-2'),
+                create_fraud_event_string(event_id_1, 'session-id-1', 'fraud-event-id-1'),
+                create_fraud_event_string(event_id_2, 'session-id-2', 'fraud-event-id-2'),
             ]
         )
 
         event_handler.store_queued_events(None, None)
 
-        self.__assert_audit_events_table_has_fraud_event_records([('sample-id-1', 'session-id-1', 'fraud-event-id-1'), ('sample-id-2', 'session-id-2', 'fraud-event-id-2')])
+        self.__assert_audit_events_table_has_fraud_event_records([(event_id_1, 'session-id-1', 'fraud-event-id-1'), (event_id_2, 'session-id-2', 'fraud-event-id-2')])
```

Comment:
Having to pass these tuples around makes me sad. I guess we need an ORM. Hrm.

Reply:
It makess me sad too, I did think of re-factoring and the above calls to `create_fraud_event_string()` so that they returned an object rather than string that we could use to pass around to the assert methods ...sort of ORM like.

Reply:
Sure, `namedtuple` perhaps, whatever's the best combination of easy and clear considering this is only test code. Happy to approve whether or not you do this, anyway.

Reply:
I'll put in a tech-debt card to re-factor these and the other tests in the module (and the `import_handler_test` module) as they all suffer the same deficiencies and some code could be moved to a shared helper.

Reply:
Cool. (As long as it goes on our board, not on the tech debt board. It can go there if it goes more gets to the end of the quarter without moving!)